public/app/frontend/src/components/breadcrumbs/breadcrumbs.html.twig

```diff
@@ -0,0 +1,42 @@
+{#
+@file
+A component to display breadcrumbs
+
+Available variables:
+    - links: array An array of links to be displayed
+        - url: string The url for the link
+        - label: string The label for the link
+example:
+    {% include '@components/breadcrumbs/breadcrumbs.html.twig' with {
+        links: [
+            {
+                url: '#',
+                label: 'Home'
+            },
+            {
+                url: '#',
+                label: 'Procedure rules'
+            },
+            {
+                url: '#',
+                label: 'Family'
+            },
+        ]
+    }%}
+#}
+
+<nav class="breadcrumbs" aria-label="Breadcrumb">
+    <ul class="breadcrumbs__list">
+        {% for link in links %}
+            {% if loop.last %}
+                <li class="breadcrumbs__item">
```

Comment:
Super minor one, as the li is the same for both conditions, it could be moved outside of the `if/else` to save a few lines.